HW_5/index.js

```diff
@@ -0,0 +1,159 @@
+}
+
+function setTasksToLocalStorage(tasks) {
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+
+function createSingleTaskElement(task) {
+    const li = document.createElement('li');
+    li.className = 'collection-item';
+
+    const textElement = document.createElement('span');
+    textElement.className = 'text-item';
+    textElement.textContent = task;
+    li.appendChild(textElement);
+
+    const editElement = document.createElement('span');
+    editElement.className = 'edit-item';
+    editElement.innerHTML = '<i class="fa fa-edit"></i>';
+    editElement.addEventListener('click', () => editOneElement(li, textElement, editElement));
+
+    li.appendChild(editElement);
+
+    const deleteElement = document.createElement('span');
+    deleteElement.className = 'delete-item';
+    deleteElement.innerHTML = '<i class="f fa-remove"></i>';
+    li.appendChild(deleteElement);
+    deleteElement.addEventListener('click', clearSingleTask);
+
+    taskList.appendChild(li);
+
+}
+
+function editOneElement(edit, textElement, editButton) {
+    const editTask = document.createElement('input');
+    textElement.remove();
+    editButton.remove();
+    editTask.setAttribute('type', 'text');
+    editTask.value = textElement.textContent;
+    editTask.addEventListener("keypress", function (event) {
+        if (event.key === "Enter") {
+            event.preventDefault();
+            editTask.remove();
+            console.log(edit.textContent)
+            textElement.textContent = editTask.value;
+            edit.insertBefore(editButton, edit.children[0]);
+            edit.insertBefore(textElement, edit.children[0]);
+
+            const nodes = Array.prototype.slice.call(taskList.children);
+            const index = nodes.indexOf(textElement.parentElement);
+            updateTaskInLocalStorage(textElement.textContent, index);
+        }
+    })
+
+    edit.insertBefore(editTask, edit.children[0]);
+    console.log(edit)
+
+}
+
+
+function renderTasks() {
+    let tasks;
+
+    if (localStorage.getItem('tasks') !== null) {
+        tasks = JSON.parse(localStorage.getItem('tasks'));
+    } else {
+        tasks = [];
+    }
+
+    tasks.forEach((task) => {
+        createSingleTaskElement(task);
+    })
+}
+
+function createTask(event) {
+    event.preventDefault();
+    if (taskInput.value.trim() === '') {
+        return;
+    }
+
+    createSingleTaskElement(taskInput.value);
+
+    storeTaskInLocalStorage(taskInput.value);
+
+    taskInput.value = '';
+}
+
+function updateTaskInLocalStorage(updatedTask, index) {
+    const tasks = getTasksFromLocalStorage();
+    tasks[index] = updatedTask;
+    setTasksToLocalStorage(tasks);
+}
+
+function storeTaskInLocalStorage(task) {
+    let tasks;
+    if (localStorage.getItem('tasks') !== null) {
+        tasks = JSON.parse(localStorage.getItem('tasks'));
+    } else {
+        tasks = [];
+    }
```

Comment:
```suggestion
let tasks = getTasksFromLocalStorage();
```

Функція getTasksFromLocalStorage ма/
є в собі цей же функціонал